papers/david_dotson/david_dotson.rst

```diff
@@ -140,7 +140,7 @@ and so we now have::
                 +-- text/
 
 Accessing paths in this way returns ``Tree`` and ``Leaf`` objects, which refer to directories and files, respectively.
-These paths may not point to directories or files that actually exist, but they can be used to create and work with ``Tree`` and ``Leaf`` elements.
+These paths may not point to directories or files that actually exist, but they can be used to create and work with these filesystem elements.
```

Comment:
"may not" is ambiguous: you don't mean "are not allowed to" but rather "don't have to"